schemas/v3.1/meta/validation.json

```diff
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://spec.openapis.org/oas/3.1/meta/validation/2019-10",
+    "$vocabulary": {
+        "https://json-schema.org/draft/2019-09/vocab/validation": true
+    },
+    "$recursiveAnchor": true,
+
+    "$comment": "This is an alternate meta-schema for the standard validation vocabulary.  It is identical to the standard validation meta-schema except that it omits exclusiveMinimum and exclusiveMaximum, which are described by the OpenAPI extensions meta-schema.",
+
+    "title": "Validation vocabulary meta-schema",
+    "type": ["object", "boolean"],
+    "properties": {
+        "multipleOf": {
+            "type": "number",
+            "exclusiveMinimum": 0
+        },
+        "maximum": {
```

Comment:
No, because their syntax hasn't changed, and the meta-schema is about syntax.

Reply:
This recommendation assumes that OAS defines it's own "validation" vocabulary and the modified keywords are all defined in the "extensions" vocabulary. Without that change, you are correct that it is not wrong as it is. It's just awkward.

However, I realize that this recommendation is backwards. If OAS defines it's own "validation" vocabulary, the best place for the modified keywords is in that vocabulary not the "extensions" vocabulary. It would be much more straightforward to have all the validation keywords together.

Reply:
@jdesrosiers OAS is **not** defining its own validation vocabulary.  Meta-schemas do not define vocabularies.  At all.  That is why the vocabulary URI is not the meta-schema URI.  Meta-schemas describe the acceptable syntax for a dialect (a collection of vocabularies- that term, "dialect", will be in the next draft).  You often want to break down meta-schemas by vocabulary because that's efficient, but there is no required correlation at all.  You could just write a single-file meta-schema for all of your vocabularies if you want to.

Reply:
> OAS is **not** defining its own validation vocabulary.

I understand that. My suggestion is that it **should** define its own validation vocabulary. My suggestion is that instead of OAS 3.1 defining one vocabulary, it defines two. One vocabulary for added keywords ("extensions") and another for it's custom version of the "validation" vocabulary.

_If_ it were done this way, the modified keywords would be part of the custom "validation" vocabulary and would therefore make the most sense in the custom "validation" meta-schema. Yes, it could be in the "extensions" meta-schema, but it would be a strange choice.

As I said, if you don't take my suggestion of making a custom "validation" vocabulary, the suggestion of moving `maximum` and `minimum` definitions can be ignored.

Reply:
@jdesrosiers whether OpenAPI decides to create its own validation vocab in the future (which I do not think it needs to do at all), the point here is moot for the simple reason that `minimum` and `maximum` work *exactly* the same in OpenAPI as they do in JSON Schema. 

Defining them twice would be pointless and redundant.

@handrews Let's resolve this thread and crack on.

Reply:
@philsturgeon No, the point is that they don't work the same. In OpenAPI, their behavior can change in the presence of `exclusiveMaximum`/`exclusiveMinimum` with a boolean value. Their meta-schema doesn't change, but their semantics do. I have to provide an OpenAPI-specific implementation of `minimum` and `maximum`. I can't use the standard implementation.

I still think one vocabulary adding semantics to another vocabulary is a bad idea (removing semantics via meta-schema is fine), but if you don't choose to make a separate validation vocabulary, I'd still consider moving `minimum` and `maximum` over (even tho it's functionally equivalent), because it helps communicate that they're not the standard keywords and that the extensions vocabulary is overriding something about their standard behavior.

But, I agree that it's time to make a decision and move on. I've shared my experience implementing this vocabulary and my opinions. I'll work with whatever is decided, whether I agree with the decision or not.

Reply:
> I still think one vocabulary adding semantics to another vocabulary is a bad idea (removing semantics via meta-schema is fine), 

This is a little confusing because the discussion keeps jumping between different threads on a PR. This PR only aims to document the current situation, not change the current situation. 

Please make a new issue to discuss the concerns and mark your concerns as resolved here so we can clear up the PR and move forward. That discussion can then be had at greater length with people who actually represent OpenAPI (which is neither myself or Henry).

Reply:
@jdesrosiers as you can see in #2219, we are now considering either changing from 3.1 to 4.0, or dropping SemVer, which would allow us to get rid of this `exclusive*` mess (and allow the next JSON Schema draft to tighten up the language around combining vocabularies).  While this problem is not the only reason the TSC is considering this change, it helped make the case stronger, so thanks for your persistence!

There's still a chance we'll stick with this so I'm not updating this PR yet, but hopefully will be able to do so after the next TSC meeting.